dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -648,6 +649,66 @@ def test_profile(self):
             self.assertAlmostEqual(expected_quartiles[1], quartiles[1])
             self.assertAlmostEqual(expected_quartiles[2], quartiles[2])
 
+    @mock.patch.multiple(NumericStatsMixin, __abstractmethods__=set(),
+                         _filter_properties_w_options=mock.MagicMock(
+                             side_effect=BaseColumnProfiler._filter_properties_w_options),
+                         create=True)
+    def test_report(self):
+        options = NumericalOptions()
+        options.max.is_enabled = False
+        options.min.is_enabled = False
+        options.histogram_and_quantiles.is_enabled = False
+        options.variance.is_enabled = False
+
+        num_profiler = NumericStatsMixin(options=options)
+
+        mock_report = dict(
+            min=1.0,
+            max=1.0,
+            median=np.nan, # default
+            mode=[np.nan], # default
+            sum=1.0,
+            mean=0, # default
+            variance=np.nan, # default
+            skewness=np.nan, # default
+            kurtosis=np.nan, # default
+            median_abs_deviation=np.nan, # default
+            stddev=np.nan, # default
+            histogram={
+                'bin_counts': np.array([1, 1, 1]),
+                'bin_edges': np.array([1.0, 2.0, 3.0, 4.0])
+            },
+            quantiles={
+                0: 2.0,
+                1: 3.0,
+                2: 4.0,
+            },
+            num_zeros=0, # default
+            num_negatives=0, # default
+            times=defaultdict(float), # default
+        )
+
+        num_profiler.match_count = 0
```

Comment:
prob only need to specify hist, sum, and variance. rest is dependent on anything

Reply:
yep